templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+{% block content %}
+    <h1>Create Manufacturer</h1>
+    <form action="" method="post" novalidate>
+    {% csrf_token %}
+    <input class="btn btn-primary" type="submit" value="Submit">
```

Comment:
it will be a good option for the user to cancel deletion , that’s why we have an extra step before deleting something